nchelpers/__init__.py

```diff
@@ -1700,15 +1705,17 @@ def _cmor_type_filename_components(
         elif self.is_other:
 
             # CAUTION: Temporary solution to a bigger problem here
-
             components.update(
-                mip_table=tres_to_mip_table and
-                          tres_to_mip_table.get(self.time_resolution, None),
                 model=self.metadata.model,
                 experiment=self.metadata.experiment,
                 geo_info=getattr(self, 'domain', None)
             )
 
+            if self.is_downscaled_gridded_obs:
+                components.update(
+                    downscaling_method=self.method_id
+                )
+
```

Comment:
Alternatively, pull the case of `self.is_downscaled_gridded_obs` up to the level of `self.is_other`, leaving `self.is_other` for last as it is a more general condition than `self.is_downscaled_gridded_obs`. 

Does that make sense? Generally speaking, we want to prefer less nesting of conditions.

Reply:
Yes it does!